homeassistant/core.py

```diff
@@ -426,6 +428,12 @@ def stop(self) -> None:
         await self.async_block_till_done()
 
         # stage 2
+        self.state = CoreState.writing_data
+        self.async_track_tasks()
```

Comment:
We're still tracking tasks here, no need to turn on.

Reply:
updated